rules/eslint/es6/walmart.js

```diff
@@ -19,6 +27,12 @@ module.exports = {
     // require method and property shorthand syntax for object literals
     "object-shorthand": 2,
     // suggest using of const declaration for variables that are never modified after declared
-    "prefer-const": 2
+    "prefer-const": 2,
+    // suggest using the spread operator instead of .apply()
+    "prefer-spread": 0,
```

Comment:
Hmm... I like having the option of either whichever makes sense.  Sometimes you want to apply a context along with the args.